### PR TITLE
Adds give user a read access on account if he has access on the cloud…

### PR DESCRIPTION
**Pull request by ashraffouda**
… space

Signed-off-by: Ashraf Fouda <ashraf.m.fouda@gmail.com>

### CODE REVIEW

**Comment by grimpy** [comment]
this is ok now when given access to the cloudspace...what about when given access to a vm?

**Comment by katia-e** [comment]
solving #1365

File: apps/cbportal/base/cloudbroker__cloudspace/methodclass/cloudbroker_cloudspace.py [hunk 1]
```diff
@@ -458,6 +458,8 @@ def deleteUser(self, cloudspaceId, username, recursivedelete, **kwargs):
             # external user, delete ACE that was added using emailaddress
             userId = username
         j.apps.cloudapi.cloudspaces.deleteUser(cloudspaceId=cloudspaceId, userId=userId, recursivedelete=recursivedelete)
+
+
```
Comment (grimpy) [inline apps/cbportal/base/cloudbroker__cloudspace/methodclass/cloudbroker_cloudspace.py]: remove these whitelines again

**Comment by ashraffouda** [comment]
added tests here https://github.com/0-complexity/openvcloud/pull/1675

**Pull request merged**